dln/vi/model.py

```diff
@@ -709,7 +710,7 @@ def forward(self, x, y=None, infos=None, temperature=0.0, cost_only=False):
                 x_stripped, temperature=temperature, max_tokens=self.p1_max_tokens
             )
 
-            self.cost += self.encoder_l2.forward_evaluate.compute_cost(x_stripped)
+            self.cost += self.forward_evaluate.compute_cost(x_stripped)
```

Comment:
Is that needed? I thought the purpose of the registry was to avoid having to track the cost like this.

Reply:
Yes, using the registry this is not required.